core/loadpoint_session.go

```diff
@@ -33,7 +33,9 @@ func (lp *Loadpoint) createSession() {
 
 	lp.session = lp.db.New(lp.chargeMeterTotal())
 
-	if vehicle := lp.GetVehicle(); vehicle != nil {
+	if lp.chargerHasFeature(api.IntegratedDevice) {
+		lp.session.Vehicle = lp.GetTitle()
```

Comment:
Technisch kann ein Integrated Device sogar ein Vehicle haben. @VolkerK62 war das nicht dein Use Case? Daher würd ich diesen Check nach dem Fahrzeug machen.

Reply:
Ja, hab ich bei mir, um Soc/Temp angezeigt zu bekommen.

Reply:
Nach https://github.com/evcc-io/evcc/pull/19544 wird das auch nicht mehr möglich sein :O

Reply:
Custom charger kann das, aber nicht das "Shelly" template, oder?

Reply:
Hab gerade von unterwegs geantwortet.
Nochmal drüber nachgedacht: 
Auch wenn ein Vehicle zugewiesen ist, ist es kein Problem, wenn in der Liste als Vehicle der Name des Loadpoint aufgeführt ist.